src/servers/Scripts/quest/classquest/ARC/ClsArc000.cpp

```diff
@@ -73,9 +73,9 @@ class ClsArc000 : public EventScript
       if( npcId != Enemy0 || npcId != Enemy1 || npcId != Enemy2 )
```

Comment:
Unless I'm missing something, the code suggested won't work.  As it is written now this if statement would always be true.  What you really need is something like:
```
 if( npcId != Enemy0 && npcId != Enemy1 && npcId != Enemy2 )
         return; 
```
or you could write:
```
if (!(npcId == Enemy0 || npcId == Enemy1 || npcId == Enemy2))
  return;
```